### ISSUE TITLE
Need to silence stderr for all ssh commands

### ISSUE THREAD

**Issue by chris-allen** [chris-allen/dream-ops#5]
Sample deploy output with an abnormal SSH configuration:

Command:
```bash
$ dream deploy solo -i ~/.ssh/test.pem -T ubuntu@1.2.3.4
```

Output:
```bash
Target: Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
ip-1-2-3-4
Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
--- Cookbook: chef-app (outdated)
...Building cookbook [chef-app]
...Deploying cookbook [chef-app]
...Syncing [repo="app" target="ubuntu@1.2.3.4"]
Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
...Running setup role [target="ubuntu@1.2.3.4"]
Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
...Running deploy role [target="ubuntu@1.2.3.4"]
Warning: Permanently added '1.2.3.4' (ECDSA) to the list of known hosts.
```

**Comment by chris-allen** [comment]
This is still a problem with `dream-ops:0.4.1`.  The `~/.ssh/config` to use:

```
UserKnownHostsFile=/dev/null
StrictHostKeyChecking=no
```

**Comment by chris-allen** [comment]
Verified fixed in `0.4.2`
